.evergreen/auth_oidc/azure/create-and-setup-vm.sh

```diff
@@ -27,8 +28,12 @@ if [ ! -f ./secrets-export.sh ]; then
 fi
 source ./secrets-export.sh
 
+echo "${AZUREOIDC_CERT}" | base64 --decode > $AZUREKMS_CERTFILE
```

Comment:
Suggest adding check for `AZUREOIDC_CERT` in the "Ensure required variables are set" block above.

Reply:
The cert is found from the vault, the only required manual variable for this test is AZUREOIDC_VMNAME_PREFIX.